authors/settings.py

```diff
@@ -84,9 +84,9 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
-DATABASE_URL = config('DATABASE_URL')
-DATABASES = {}
-DATABASES['default'] = dj_database_url.config(conn_max_age=600, ssl_require=True)
+DATABASES = {
```

Comment:
@samuelbwambale How is this change relevant to the issue that your PR seeks to address?

Reply:
@lym I was not able to register a user locally with the previous database settings. I changed the settings to test that the feature works as I expected. The app functions properly both locally and on Heroku

Reply:
@samuelbwambale Did the app not work for anyone till this point?

Reply:
The issue started after I rebased from develop this morning. @MuhanguziDavid experienced the same while working on JWT Authentication.
The app was starting normally but on registering a user, I was getting an error `raise ImproperlyConfigured("settings.DATABASES is improperly configured. "`

Reply:
The proper way to do this would've been to create two separate PRs; the first one fixing the database configuration issue and the second implementing your story. I cannot merge this PR as it stands.

Reply:
Thanks @lym.  [#160897544](https://www.pivotaltracker.com/story/show/160897544) has been raised on PT board to fix the database configuration issue

Reply:
@lym Issue fixed